x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_status_dropdown.tsx

```diff
@@ -55,34 +57,64 @@ const COMMON_SNOOZE_TIMES: Array<[number, SnoozeUnit]> = [
   [1, 'd'],
 ];
 
+const PREV_SNOOZE_INTERVAL_KEY = 'triggersActionsUi_previousSnoozeInterval';
+const usePreviousSnoozeInterval: (p?: string | null) => [string | null, (n: string) => void] = (
+  propsInterval
+) => {
+  const intervalFromStorage = localStorage.getItem(PREV_SNOOZE_INTERVAL_KEY);
```

Comment:
I am wondering if we should allow different `SNOOZE_INTERVAL_KEY` for different solutions, maybe let's see if we get this requirement in the future.

Reply:
match with this comment https://github.com/elastic/kibana/pull/130205#discussion_r850737730